.github/workflows/slash-commands.yml

```diff
@@ -27,6 +27,7 @@ jobs:
             test-performance
             publish-java-cdk
             connector-performance
+            approve-regression-tests
```

Comment:
I always find difficult to understand slash commands as these commands must match `<workflow-name>-command` ...

Reply:
Yes that wasn't obvious to me either.